tokenserver/assignment/sqlnode/sql.py

```diff
@@ -224,10 +225,16 @@ def __init__(self, sqluri, create_tables=False, pool_size=100,
         else:
             from tokenserver.assignment.sqlnode.schemas import get_cls
 
+        self._spanner_node_id = spanner_node_id
+        self._migrate_new_user_percentage = migrate_new_user_percentage
+        if self._migrate_new_user_percentage > 0:
+            if self._spanner_node_id is None:
+                raise ValueError("Must set spanner_node_id when migrating "
+                                 "users to spanner")
```

Comment:
@pjenvey would not approve. ;)

Reply:
Well now I'm curious as to why :-)

Reply:
He prefers that exceptions like this be package specific and not use existing exception types when possible. (So, something like `ConfigException` rather than `ValueError`) I've kinda learned to agree since using a custom allows you a good deal more flexibility in the future with things like error reporting and potential mitigation.

Reply:
That's fair; come to think of it I usually prefer to use `RuntimeError` or `AssertionError` for things like this, since they're less likely to be accidentally caught and ignored.